### DISCUSSION TITLE
What does that mean abandoned production in GitHub for Gauge

### DISCUSSION THREAD

**Discussion by zbrea** [getgauge/gauge#2154] [General]
Dear gauge team !
I gave up Cucumber for Gauge :)
I managed to convince my company to use Gauge, we are going to use it for multiple projects, we have already started the automation. Hope all is well in the future.
What does that mean abandoned production ? 
Reassure me, that the project is not abandoned ! :(
![image](https://user-images.githubusercontent.com/59898499/140653861-219e851c-45a4-4948-9ae3-1438012f23f6.png)

**Comment by zabil** [comment]
Where do you see this? Can you please share a full link to the page?

**Reply by zbrea** [reply]
https://github.com/getgauge/gauge

![image](https://user-images.githubusercontent.com/59898499/140654421-bf75a326-8faa-426b-bd00-233285f07515.png)

**Reply by zabil** [reply]
It's not related to the status of the project but something related to github deployments and github pages as Gauge does not use github pages.

**Comment by zbrea** [comment]
I see the same problem for :
gauge-java (https://github.com/getgauge/gauge-java)
gauge-vscode (https://github.com/getgauge/gauge-vscode)
gauge-gradle-plugin (https://github.com/getgauge/gauge-gradle-plugin)
gauge-maven-plugin (https://github.com/getgauge-contrib/gauge-maven-plugin)

**Comment by zabil** [comment]
Please refer https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment

For more information about github environments.

**Comment by zbrea** [comment]
Thank you for your positive response, it reassured me :)
Gauge is the best!
Better than SpecFlow and Cucumber
